duckduckgo/macos-browser
# Geoswitching design and copy review feedback

Task/Issue URL: https://app.asana.com/0/0/1206417809840737/f
Design review Task/Issue URL: https://app.asana.com/0/0/1206382695809291/f
Copy review Task/Issue URL: https://app.asana.com/0/38424471409662/1206382794710141/f

**Description**:
Design and copy review. Includes:
- [x] The list seems to load each time when I open it. I thought we'd cache it after opening it for the first time. Is this something we need to add?
- [x] Could we make the dialog slightly larger vertically (max-height?) so we use more screen real estate?
- [x] For the US city drop-down, can we make the width of it larger so it can show the full city names?
- [x] The country label text-color is slightly off. It should be the same color as the IP address.
- [x] I think we could actually remove the spinner, the list loads pretty fast.

**Steps to test this PR**:
1. Enabled Network Protection
2. Go to the various Status View UIs, the VPN Locations sheet and the VPN preferences screen and check that this feedback has been addressed.

—
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
